service/src/main/java/bio/terra/workspace/common/logging/WorkspaceActivityLogHook.java

```diff
@@ -353,30 +355,28 @@ private void maybeLogForSyncGcpIamRolesFlight(
     }
   }
 
-  private void maybeLogUpdateControlledResourceRegionFlight(
+  private void maybeLogUpdateControlledResourceFieldsFlight(
       FlightContext context, OperationType operationType, String userEmail, String subjectId) {
     if (!context.getFlightStatus().equals(FlightStatus.SUCCESS)) {
       return;
     }
     FlightUtils.validateRequiredEntries(
-        context.getWorkingMap(),
-        CONTROLLED_RESOURCE_ID_TO_REGION_MAP,
-        CONTROLLED_RESOURCE_ID_TO_WORKSPACE_ID_MAP);
-    Map<UUID, String> resourceIdToRegionMap =
-        context.getWorkingMap().get(CONTROLLED_RESOURCE_ID_TO_REGION_MAP, new TypeReference<>() {});
+        context.getWorkingMap(), CONTROLLED_RESOURCE_ID_TO_WORKSPACE_ID_MAP);
+
     Map<UUID, String> resourceIdToWorkspaceIdMap =
-        context
-            .getWorkingMap()
-            .get(CONTROLLED_RESOURCE_ID_TO_WORKSPACE_ID_MAP, new TypeReference<>() {});
+        Preconditions.checkNotNull(
+            context
+                .getWorkingMap()
+                .get(CONTROLLED_RESOURCE_ID_TO_WORKSPACE_ID_MAP, new TypeReference<>() {}));
 
-    for (var resourceId : resourceIdToRegionMap.keySet()) {
+    for (var pair : resourceIdToWorkspaceIdMap.entrySet()) {
```

Comment:
Nit: by convention, we do not use `var` unless it is obvious what the type is; e.g., a builder or a `new`.